src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java

```diff
@@ -0,0 +1,925 @@
+    @LuaFunction(mainThread = true)
+    public final MethodResult importItem(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected(0);
+
+        IItemHandler inventory = InventoryUtil.getHandlerFromDirection(arguments.getString(1), owner);
+
+        if (inventory == null) {
+            inventory = InventoryUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (inventory == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return importToRS(arguments, inventory);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult exportItem(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected(0);
+
+        IItemHandler inventory = InventoryUtil.getHandlerFromDirection(arguments.getString(1), owner);
+
+        if (inventory == null) {
+            inventory = InventoryUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (inventory == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return exportToChest(arguments, inventory);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult importFluid(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected(0);
+
+        IFluidHandler handler = FluidUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null) {
+            handler = FluidUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (handler == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return importToRS(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult exportFluid(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected(0);
+
+        IFluidHandler handler = FluidUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null) {
+            handler = FluidUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (handler == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return exportToTank(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult importChemical(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected(0);
+
+        if (!APAddons.refinedStorageMekanismLoaded)
+            return MethodResult.of(0);
```

Comment:
there is no information about that why nothing is imported/exported

Reply:
a shit forgot about that